packages/core/src/physics/shape/ColliderShape.ts

```diff
@@ -101,6 +116,17 @@ export abstract class ColliderShape {
     this._nativeShape.setPosition(this._position);
   }
 
+  /**
+   * Set the local rotation of collider shape
+   * @param x - Radian of yaw
+   * @param y - Radian of pitch
+   * @param z - Radian of roll
+   */
+  setRotation(x: number, y: number, z: number): void {
+    this._rotation.set(x, y, z);
+    this._nativeShape.setRotation(this._rotation);
+  }
```

Comment:
Suggest use rotion callback to dispose `rotation` and `position` change. developer modify signle axis is  common